pkg/cli/cmd/templates/install.go

```diff
@@ -353,7 +353,6 @@ func (i *tmplInstaller) runTemplateTests() error {
 	for _, v := range tests {
 		command := directory.TestExecCmd{
 			File:    v,
-			NoColor: false,
```

Comment:
Doesn't this mean that `topaz templates install` would only run the directory tests?
Don't we need to run `authorizer.TestExecCmd` too?

Reply:
Yes, will add